src/fields/BaseOptionsField.php

```diff
@@ -239,11 +239,12 @@ public function normalizeValue($value, ElementInterface $element = null)
         }
 
         if (is_string($value) && (
-                $value === '' ||
                 strpos($value, '[') === 0 ||
```

Comment:
Removing this allows `$value` to remain `''` instead of being coerced to `null`.